crates/red_knot_python_semantic/resources/mdtest/terminal_statements.md

```diff
@@ -635,6 +635,5 @@ def _(cond: bool):
         if True:
             return
 
-    # TODO: Literal["a"]
-    reveal_type(x)  # revealed: Literal["a", "b"]
+    reveal_type(x)  # revealed: Literal["a"]
```

Comment:
The prose paragraph above this test needs updating.

Reply:
Done